### PR TITLE
fix: add KaranChandekar

### LINKED ISSUE #1191: Karan Chandekar
## Profile Screenshot

github-profile

### PR DESCRIPTION
**Pull request by KaranChandekar**
resolves #1191 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by KaranChandekar** [comment]
Please review my pull request!

File: profiles/KaranChandekar.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/93200960/174483769-0ec591f5-8438-4c97-8304-052a21f41254.png",
+    "issueId": 1191,
+    "name": "Karan Chandekar",
+    "username": "KaranChandekar"
+}
+
```
Comment (Panquesito7) [inline profiles/KaranChandekar.json:7]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Amazing! Thank you. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant

**Comment by KaranChandekar** [comment]
@Panquesito7 It's mine pleasure!

**Pull request merged**